content/docs/command-reference/checkout.md

```diff
@@ -151,7 +151,7 @@ baseline-experiment     <- First simple version of the model
 bigrams-experiment      <- Uses bigrams to improve the model
 ```
 
-We can now just run `dvc checkout` that will update the most recent `model.pkl`,
+We can now run `dvc checkout` that will update the most recent `model.pkl`,
```

Comment:
This was also not condescending. "Just" means only: something you have to do once.

But the sentence was a little weird anyway so let's use this instead:

"We can now run `dvc checkout` to update the most ..."

Reply:
> I also don't think this was condescending. "Just" means "only": something you perform once or with little effort, and then you're done.

As a doc writer, you are suggesting to the user this command is so simple. Do you actually need to do that?
I am making the change as you mentioned.